fvm/src/executor/default.rs

```diff
@@ -19,14 +19,7 @@ use crate::gas::{GasCharge, GasOutputs};
 use crate::kernel::{ClassifyResult, Context as _, ExecutionError, Kernel};
 use crate::machine::{Machine, BURNT_FUNDS_ACTOR_ADDR, REWARD_ACTOR_ADDR};
 
-/// The core of the FVM.
-///
-/// ## Generic types
-/// * B => Blockstore.
-/// * E => Externs.
-/// * K => Kernel.
-//
-// If the inner value is `None` it means the machine got poisend and is unusable.
+// If the inner value is `None` it means the machine got poisoned and is unusable.
```

Comment:
Hm. We should document this... but the documentation was wrong anyways (I probably copied it from the Machine).